engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -359,6 +359,9 @@ public void setHostAllocators(final List<HostAllocator> hostAllocators) {
             Integer.class, "vm.job.report.interval", "60",
             "Interval to send application level pings to make sure the connection is still working", false);
 
+    static final ConfigKey<Boolean> HaVmRestartHostUp = new ConfigKey<Boolean>("Advanced", Boolean.class, "ha.vm.restart.hostup", "true",
```

Comment:
I would expect to see the new ConfigKey be added to the return of getConfigKeys()

Reply:
Agreed, not sure how that got missed here.

Reply:
fixed